meet_balls_app/tests.py

```diff
@@ -1,3 +1,6 @@
-# from django.test import TestCase
+class TestUi:
 
-# Create your tests here.
+    def test_home_page_extends_base(self, client):
```

Comment:
can you please validate here that you see the login and create another test where you have a user that is already after login and check you see the logout please

Reply:
I don't see the change in the PR did you push?

Reply:
I forgot to amend the cahnges